script/functions.sh

```diff
@@ -11,12 +11,6 @@ export JRUBY_OPTS=${JRUBY_OPTS:-"--server -Xcompile.invokedynamic=false"}
 SPECS_HAVE_RUN_FILE=specs.out
 MAINTENANCE_BRANCH=`cat maintenance-branch`
 
-# Don't allow rubygems to pollute what's loaded. Also, things boot faster
-# without the extra load time of rubygems. Only works on MRI Ruby 1.9+
-if is_mri_192_plus; then
-  export RUBYOPT="--disable=gem"
-fi
-
```

Comment:
Ok. Notice that this prevents installation of latest `nokogiri`, at least on my system. Hopefully they merge [this patch](https://github.com/sparklemotion/nokogiri/pull/1393) which fixes the issue (and makes setting this useless).